src/components/grapher/Grapher.vue

```diff
@@ -285,4 +308,9 @@ export default Vue.extend({
     opacity: 0.5;
   }
 }
+.grapher--dottext {
+  fill: $black;
+  font-size: 2px;
+  text-anchor: middle;
```

Comment:
nit: Can we change the text-anchor to `start` so that the left side of the label is always in the same place?
```suggestion
  text-anchor: start;
```

Reply:
Good idea, and I'll move the label a little to the left to match.